### PR TITLE
Added Roadmaps for Web Development

### LINKED ISSUE #156: Addition of Roadmap for Web Development

### PR DESCRIPTION
**Pull request by StutiRajput07**
This closes #156 

- `Web development - Roadmap for Web Development`

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: Web Development/README.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
 ## Table of Contents
 
 - [Overview](#overview)<br>
+
+
+- [Roadmap for Web Development](roadmaps)
+
+  - [Front-End Development Roadmap](#front-end-development)
+  - [Back-End Development Roadmap](#backend-development)
+  - [Full-Stack Development Roadmap](#full-stack-development)
```
Comment (jfmartinz) [inline Web Development/README.md:12]: Create a seperate table for each field, this is the syntax:

```suggestion
### Front-End Development Roadmap
<table>
<tr>
   <td><a href="URL_here">Resource Name</a></td>
   <td>URL Description</td>
</tr>
</table>

### Back-End Development Roadmap
<table>
<tr>
   <td><a href="URL_here">Resource Name</a></td>
   <td>URL Description</td>
</tr>
</table>

And so on
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by StutiRajput07** [comment]
I have made the requested changes.

File: Web Development/README.md [hunk 2]
```diff
@@ -5,13 +5,7 @@
...
 
 
-- [Roadmap for Web Development](roadmaps)
-
-  - [Front-End Development Roadmap](#front-end-development)
-  - [Back-End Development Roadmap](#backend-development)
-  - [Full-Stack Development Roadmap](#full-stack-development)
-
-
+- [Roadmap](roadmaps)
```
Comment (jfmartinz) [inline Web Development/README.md:8]:
```suggestion
- [Roadmap](roadmaps)
      - [Front-End Development Roadmap](#front-end-development)
      - [Back-End Development Roadmap](#backend-development)
      - [Full-Stack Development Roadmap](#full-stack-development)
```

File: Web Development/README.md [hunk 3]
```diff
@@ -49,11 +43,43 @@
...
-- [Roadmap for Back-End Development](https://roadmap.sh/backend)
-- [Roadmap for Full Stack Development](https://roadmap.sh/full-stack)
+#### Front-End Development 
+<table width="100%">
+    <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+<tr>
+   <td><a href="https://roadmap.sh/frontend"> Roadmap </a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:55]:
```suggestion
   <td><a href="https://roadmap.sh/frontend"> Frontend</a></td>
```

File: Web Development/README.md [hunk 4]
```diff
@@ -49,11 +43,43 @@
...
+</table>
+
+#### Back-End Development 
+<table width="100%">
+    <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+<tr>
+   <td><a href="https://roadmap.sh/backend">Roadmap</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:67]:
```suggestion
   <td><a href="https://roadmap.sh/backend">Backend</a></td>
```

File: Web Development/README.md [hunk 5]
```diff
@@ -49,11 +43,43 @@
...
+</table>
+
+#### Full-Stack Development 
+<table width="100%">
+    <tr>
+    <th>Resource Name</th>
+    <th>Description</th>
+  </tr>
+<tr>
+   <td><a href="https://roadmap.sh/backend">Roadmap</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:79]:
```suggestion
   <td><a href="https://roadmap.sh/backend">Full-Stack</a></td>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Please change this before I merged this PR. Thank you for your patience! 👍

**Comment by StutiRajput07** [comment]
Done, please check.

**Comment by StutiRajput07** [comment]
@jfmartinz Please display the recent changes which you want to request,as I am unable to see it.

**Review by jfmartinz — CHANGES_REQUESTED** [review]
The only problem that keeping me from not merging this is the URL of the fullstack roadmap, please check. Change it and then I will merged it as soon as you changed this, thank you :)

**Comment by StutiRajput07** [comment]
Fixed!

**Pull request merged**